core/src/main/scala/ManifestValidator.scala

```diff
@@ -91,7 +91,7 @@ object ManifestValidator {
     units: List[UnitDef],
     whiteList: Option[ProxyPortWhitelist]): Valid[Unit] = {
 
-    // NOTE: aws allows elb names to be at most 32 charasters. Nelson generates a name for
+    // NOTE: aws allows nlb names to be at most 32 charasters. Nelson generates a name for
```

Comment:
Is this accurate for NLB? I expect it is, but lets check

Reply:
Yup this is still true.